bitsail-connectors/connector-redis/src/main/resources/redis-type-converter.yaml

```diff
@@ -0,0 +1,8 @@
+engine.type.to.bitsail.type.converter:
```

Comment:
this file is unnecessary because we can use the default way to do this string string conversion

Reply:
I have changed it to the string string conversion through BitSailType.
https://github.com/bytedance/bitsail/pull/43#discussion_r1016081812